rs/execution_environment/benches/summarize-results.sh

```diff
@@ -43,8 +44,8 @@ baseline_commit=$(git rev-list --abbrev-commit -1 HEAD "${BASELINE_FILE}" | head
 min_commit=$(git rev-list --abbrev-commit -1 HEAD | head -c 9)
 diff=$((diff = (total_new - total_baseline) * 100 * 10 / total_baseline))
 diff=$(echo "${diff}" | awk '{ print $0^2 <= (2 * 10)^2 ? 0 : $0 / 10 }')
-total_new_s=$((total_new / 1000 / 1000))
-printf "    = ${baseline_commit}..${min_commit}: ${NAME}: total time: ${total_new_s} s "
+total_new_ms=$((total_new / 1000 / 1000))
```

Comment:
also here -- is it milli- or micro- second?

Reply:
The original benchmark results are in ns, so in both cases it was a bug I just noticed.

Reply:
NIT: rename `new_result` to `new_result_ns` and `total_new` to `total_new_ns` to make it clear why /1000/1000 turn it to ms.